.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,56 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
 
-- Is it feasible to build and maintain with our team and runway?
-
 - Can this make us move faster?
 
-- Does this increase quality and not technical debt?
+- Does this increase quality? 
+
+- Does this add technical debt?
+
+- Is it feasible to build and maintain with our current team and runway?
 
-- Will this bring us more Collective and financial contributors?
+- Is this problem already solved elsewhere or is it based on insight or research?
 
-- Is this already solved elsewhere or is it our unique contribution?
+- How will we measure our success? Do we have a baseline today that we can compare this against?
 
-## Roadmap & Timeframe
-(Key milestones or phases and estimated delivery dates, including specs, design, feature development, feedback, marketing/comms, and documentation)
+- How well do we understand the complexity of the problem, feature to be added or user need to be addressed? 
```

Comment:
I like the estimation here, but I wonder whether this might hold us back from documenting project proposals as there's a lot of weight to doing this (and weight we shouldn't take on alone) so I've introduced some lighter weight guidance here and offloaded the heavier estimation stuff to part of kickoff/scoping